lib/components/views/FavoritesScreen.tsx

```diff
@@ -0,0 +1,108 @@
+import React, {useEffect, useState} from 'react';
+import { SafeAreaView, View, Text, FlatList, TouchableOpacity } from 'react-native';
+import { useFavoriteMovies, useUnfavoriteMovie } from "../../api/movies.ts";
+import styles from "../../styles/styles.ts"
+import {FavoriteMovie} from "../../types/movie.ts";
+
+function FavoriteMovies() {
+    const {
+        data: favorites,
+        isLoading,
+        error,
+        refetch } = useFavoriteMovies();
+    const { mutateAsync: unFavoriteMovieAsync, isLoading: isUnFavoriting } = useUnfavoriteMovie();
+    const [removingMovies, setRemovingMovies] = useState<{ [key: string]: boolean }>({});
+
+
+    async function strip(input : string){
```

Comment:
since you are using this function on multiple places you can create this helper function